dask/array/core.py

```diff
@@ -610,13 +610,27 @@ def map_blocks(
             'chunk-shape': (100,),
             'dtype': dtype('float64')}}
 
-    For each argument and keyword arguments that are dask arrays (the positions
-    of which are the first index), you will receive the shape of the full
-    array, the number of chunks of the full array in each dimension, the chunk
-    location (for example the fourth chunk over in the first dimension), and
-    the array location (for example the slice corresponding to ``40:50``). The
-    same information is provided for the output, with the key ``None``, plus
-    the shape and dtype that should be returned.
+    The keys to the ``block_info`` dictionary indicate which is the input and
+    output Dask array:
+    - ``block_info[0]`` refers to the input Dask array, and contains information
```

Comment:
(first) input Dask array

Reply:
Maybe take away "and contains information about the input chunks." since technically there's also info about the dask array, and anyways this is explained further down.

Reply:
Suggestions applied, thank you